### ISSUE TITLE
Zelda OoT segfault with Glide

### ISSUE THREAD

**Issue by richard42** [mupen64plus/mupen64plus-user-issues#581]
Originally reported on Google Code with ID 582

```
Describe your system:
 - Operating System (be specific): openSUSE 12.3 (x86_64)
 - Machine type (32-bit or 64-bit): 64-bit AMD fx-8350 Vishera
 - Mupen64Plus version: mupen64plus-bundle-linux64-2.0
 - Plugins used: mupen64plus-video-glide64mk2.so

Describe the problem:

Repeatable crash after releasing some bugs in the water temple temple (no pun intended)


Steps to reproduce:

Load Zelda OoT with the provided save state
Press the Down C button to release bugs
Segfault
```

Reported by `ZanQdo` on 2013-11-13 19:22:09

<hr>
- _Attachment: [Legend of Zelda, The - Ocarina of Time (U) (V1.0) [!].st0](https://storage.googleapis.com/google-code-attachments/mupen64plus/issue-582/comment-0/Legend of Zelda, The - Ocarina of Time %28U%29 %28V1.0%29 [!].st0)_

**Comment by richard42** [comment]
```
Just because I forgot to mention it here. It doesn't crash here when I test it with
the Debian builds. But good job with the savestate

Can you please compile the glide64mk2 plugin with the make parameters

 OPTFLAGS="-O2" DEBUG=1

and run it with gdb (please create a backtrace where it crashes). Maybe also try to
build it with OPTFLAGS="-O0" to get a slightly better backtrace
```

Reported by `sven@narfation.org` on 2013-12-03 20:50:58

**Comment by richard42** [comment]
```
Great news, I compiled from source and it seems the problem is gone!

Thanks for the suggestion.

Zan
```

Reported by `ZanQdo` on 2013-12-04 04:52:28

**Comment by richard42** [comment]
```
Ok, so it the same "bad build" problem as in 581. I'll merge both bugs together.
```

Reported by `sven@narfation.org` on 2013-12-04 08:23:19
- **Status changed**: `Duplicate`
- **Merged into**: #581
